lib/package.gi

```diff
@@ -2127,6 +2216,22 @@ InstallGlobalFunction( ValidatePackageInfo, function( info )
     fi;
     return true;
     end;
+    
+    TestOption( record, "IsMetaPackage", IsBool, "a bool" );
+
+    if IsBound( record.IsMetaPackage ) then
+      is_metapackage := record.IsMetaPackage;
+    else
+      is_metapackage := false;
+    fi;
```

Comment:
optional: `is_metapackage := IsBound( record.IsMetaPackage )  and IsBound( record.IsMetaPackage );` (resp. `hasSubpackages := ...`